lib/identity/index.js

```diff
@@ -89,7 +89,7 @@ export default class Identity extends CryptoConsumer {
     return this.constructor.StorageClient
   }
 
-  async register(username, password, token) {
+  async register(username, password, token, email) {
```

Comment:
Is email a required param?  I would think most times the username will be an email address so wondering if it's a good idea to consolidate this?

Reply:
doing it as two give additional flexibility -- the exact signature can change. I agree that most of the time email and username will be the same, but in the event they aren't, they need to be managed separately. I can even have the method check if email is defined and copy over username to email, meaning if the username is email, the parameter does not need to get sent.